src/test/resources/fake-template-repo/code/graphql/example.js

```diff
@@ -0,0 +1,57 @@
+const { makeAugmentedSchema, inferSchema } = require("neo4j-graphql-js");
```

Comment:
The template GraphQL example should be more like this example (using the new `@neo4j/graphql` library and using predefined typedefs instead of inferred): https://github.com/neo4j-graph-examples/recommendations/blob/main/code/graphql/example.js

the placeholder piece is [here](https://github.com/neo4j-graph-examples/recommendations/blob/main/code/graphql/example.js#L10-L77) Maybe change this to:

```
const typedefs =  /* GraphQL */ `
type Movie {title: String}
`
```

for the template, then just the `type Movie {title: String}` bit is replaced with the contents of `schema.graphql`

Reply:
oh maybe I misunderstood the purpose of this file. Is this meant to be the GraphQL code sample template file?

Reply:
The test is indeed confusing, there should not be any GraphQL sample in the template repository. Let me fix that.

Reply:
Something I forgot to mention: this is only for testing purposes, this file is not actually used beyond that.